ios/FluentUI/Navigation/Views/LargeTitleView.swift

```diff
@@ -354,6 +386,18 @@ class LargeTitleView: UIView {
             return !arrangedSubview.isHidden
         })
     }
+
+    // MARK: - TwoLineTitleViewDelegate
+
+    func twoLineTitleViewDidTapOnTitle(_ twoLineTitleView: TwoLineTitleView) {
+        guard twoLineTitleView == self.twoLineTitleView else {
+            return
+        }
+        guard respondsToTaps else {
```

Comment:
can the two guard statements be merged? Something like:
```
 guard twoLineTitleView == self.twoLineTitleView, respondsToTaps else {
          return;
}
```

Reply:
Is that best practice for checks that aren't closely related? Whether we're receiving a notification from the correct view and whether or not we respond to user taps seem like two different questions.

Reply:
I think we typically just group them regardless of context